schema/organization/Person.schema.yaml

```diff
@@ -2,27 +2,28 @@ title: Person
 '@id': schema:Person
 $extends: ../Thing.schema.yaml
 role: secondary
-status: unstable
+status: stable
```

Comment:
I propose that when we have finished these review on each type that we change the status to `stable`